### PR TITLE
Update for Laravel 6

### PR DESCRIPTION
**Pull request by michaeldyrynda**
We need to account for the fact that the default Bootstrap preset has been extracted to a separate package in laravel/ui and install this, before being able to run the Inertia.js preset over the top of it.

### CODE REVIEW

**Comment by reinink** [comment]
Curious, why do we even need the Bootstrap stuff for Inertia?

**Comment by michaeldyrynda** [comment]
The default views use Bootstrap because that’s what was included with Laravel when the initial version was built.

They’d probably otherwise not be very exciting, _or_ we’d have to make a decision on a) what to style them with or b) whether to include them at all.

**Comment by michaeldyrynda** [comment]
@reinink I've scrapped Bootstrap entirely, pulling in Tailwind (CDN version, to keep dependencies as light as possible).

image

I also found a way (per-Telescope) to inject a service provider into the `config/app.php` file, in order to setup default [asset versioning](https://inertiajs-new.netlify.com/asset-versioning) and [shared data](https://inertiajs-new.netlify.com/shared-data).

File: src/inertiajs-stubs/providers/InertiaJsServiceProvider.stub [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Session;
+use Illuminate\Support\ServiceProvider;
+use Inertia\Inertia;
+
+class InertiaJsServiceProvider extends ServiceProvider
+{
+    public function register()
+    {
+        $this->registerInertia();
```
Comment (reinink) [inline src/inertiajs-stubs/providers/InertiaJsServiceProvider.stub]: Any reason to not just inline this into the `register()` function? Not sure I see the benefit of the extra method here.

**Review by reinink — APPROVED** [review]
Love it! Using the CDN is a great way of handing this! 👌

File: src/inertiajs-stubs/providers/InertiaJsServiceProvider.stub (hunk 1 above)
Comment (michaeldyrynda) [reply src/inertiajs-stubs/providers/InertiaJsServiceProvider.stub]: No specific reason; removed!

**Pull request merged**